browser.js

```diff
@@ -25,6 +25,12 @@ module.exports = {
       'in client-side build of Browserslist')
   },
 
+  loadFeature: function loadFeature () {
+    throw new BrowserslistError(
+      'Feature statistics are not supported ' +
```

Comment:
`statistics`is a wrong word copied from `loadCountry`. `supports query is not …`

Reply:
Good point, fixed.